openslides/core/projector.py

```diff
@@ -139,3 +139,10 @@ class Message(ProjectorElement):
     def check_data(self):
         if self.config_entry.get('message') is None:
             raise ProjectorException('No message given.')
+
+
+class ListOfSpeakersOverlay(ProjectorElement):
+    """
+    List of speakers (last, current, next) on the projector. Rendered as overlay.
+    """
+    name = 'core/speakeroverlay'
```

Comment:
The *idea* of the "cannot see users", in my opinion, is *not* that I cannot see who's elected. I just should not get the list of all names.

I think putting a "name" field nexrt to the user_id in poll.option and agenda_item.speakers would be a good workaround. 
Pro: you only get the names of the users in question; you don't always have to query Users if all you want is a name (we can avoid some monsters à la `User.find(assignment.agenda_item.speaker.id).get_full_name` ),
Con: The name can get out of sync with the user (which is already the case with resolve); you'll add one more variable to "poll" and "agenda", which already carry around quite some stuff.